multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/util/OperationTimeAggregatorTest.java

```diff
@@ -0,0 +1,108 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
+import org.cloudfoundry.multiapps.controller.process.flowable.FlowableFacade;
+import org.flowable.engine.HistoryService;
+import org.flowable.engine.ProcessEngine;
+import org.flowable.engine.history.HistoricActivityInstance;
+import org.flowable.engine.history.HistoricActivityInstanceQuery;
+import org.flowable.engine.history.HistoricProcessInstance;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class OperationTimeAggregatorTest {
+
+    private static final String CORRELATION_ID = "id";
+
+    @Mock
+    private FlowableFacade flowableFacade;
+    @InjectMocks
+    private OperationTimeAggregator operationTimeAggregator;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    void testOperationTimeWithNoSubProcesses() {
+        //maybe split this test into multiple?
+        when(flowableFacade.getHistoricSubProcessIds(CORRELATION_ID)).thenReturn(new ArrayList<>());
+        doReturn(createProcess()).when(flowableFacade).getHistoricProcessById(CORRELATION_ID);
+
+        ProcessEngine processEngine = Mockito.mock(ProcessEngine.class);
+        Mockito.when(flowableFacade.getProcessEngine())
+               .thenReturn(processEngine);
+        HistoryService historyServiceMock = Mockito.mock(HistoryService.class);
+        Mockito.when(processEngine.getHistoryService())
+               .thenReturn(historyServiceMock);
+        HistoricActivityInstanceQuery historicActivityInstanceQueryMock = Mockito.mock(HistoricActivityInstanceQuery.class);
+        Mockito.when(historyServiceMock.createHistoricActivityInstanceQuery())
+               .thenReturn(historicActivityInstanceQueryMock);
+        Mockito.when(historicActivityInstanceQueryMock.processInstanceId(CORRELATION_ID))
+               .thenReturn(historicActivityInstanceQueryMock);
+        doReturn(mockProcessActivities("serviceTask", 2, 100))
```

Comment:
Use `Mockito.when` instead of `doReturn`.

Reply:
Done